docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -3068,18 +3020,26 @@
    "execution_count": 45,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ashwin/workspace/rapids/cudf/python/cudf/cudf/core/indexed_frame.py:2271: FutureWarning: append is deprecated and will be removed in a future version. Use concat instead.\n",
```

Comment:
@shwina Should we fix/remove the cell input that generated this warning, or possibly remove the user path from the output?

I wrote a pre-commit hook called [nb-strip-paths](https://github.com/bdice/nb-strip-paths) before joining NVIDIA that can help prevent user paths from being committed to notebooks (MIT license, but no existing SWIPAT). It would convert `/home/ashwin/workspace/rapids/cudf/python/cudf/cudf/core/indexed_frame.py` to `python/cudf/cudf/core/indexed_frame.py` if run on your local machine when committing.

Reply:
That's cool! This would be nice to have. But probably a bit late in the release for *right now*.

Reply:
Yup! I agree. Just wanted to mention it for the future.

Reply:
We still might want to fix the cell input that generated the warning.

Reply:
Fix'd